components/list/demo/list-item-sample.js

```diff
@@ -0,0 +1,52 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ListItemCheckboxMixin } from '../list-item-checkbox-mixin.js';
+//import { ListItemDragMixin } from '../list-item-drag-mixin.js';
+
+class ListItemSample extends ListItemCheckboxMixin(LitElement) {
+	// TODO: Role and breakpoints to live elsewhere
+	static get properties() {
+		return {
+			href: { type: String }
+		};
+	}
+
+	// TODO: Placeholder grid styles to be replaced with list-item-generic
+	static get styles() {
+		return [ (super.styles ? super.styles : css``), css`
+			.list-grid {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] 40px
+					[control-start outside-control-end] 40px
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			[slot="content"]{
+				grid-column: content-start / content-end;
+				grid-row: 1 / 2;
+			}
+			::slotted([slot="actions"]) {
+				grid-column: actions-start / actions-end;
+				grid-row: 1 / 2;
+				z-index: 4;
+			}
+		`];
+	}
+	render() {
+		return html`
+			<div class="list-grid">
+				${ this._renderDragHandle ? this._renderDragHandle() : html`
+					<div>=</div>
+				` }
+				${ this._renderCheckbox() }
+				<div slot="content">
```

Comment:
Thanks Zina.  This makes more sense to me now.  I _think_ the `slot="content"` should become `class="d2l-list-item-content"` though, since there's no named slot `content`.

Reply:
There will be a named content slot similar to the `control` and `control-action` slots. See this PR for context: https://github.com/BrightspaceUI/core/pull/506/files

I can change it back if it makes more sense to do so in this PR

Reply:
Ok I see how this is intended to eventually be used.  I think it would have been ok if we'd included the generic "item" in this PR, but let's get this in.

Reply:
Yeah, I think in retrospect I should have done the grid PR first.

Reply:
I think the grid allows for us to use classes instead of slots. So one default slot and people can pick from the predefined classes to place things. If that make sense.